assets/src/dashboard/app/serializers/storyPreview.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export default function reshapeStoryPreview(storyProps = {}) {
+  const story = {
+    ...storyProps,
+    date: storyProps.date || '',
```

Comment:
It looks like the `OutputStory` component expects `story.date` and `story.modified` to be defined and since this is only for a preview, I wonder if we could just assign Date.now if the value is missing for each just so the component doesn't complain in the console:
![image](https://user-images.githubusercontent.com/40646372/88960259-1b962080-d258-11ea-8544-d24dcd886bcc.png)

Reply:
This should be resolved :) fixing the date/modified led to some more exposed warnings, so I added those in too - should be a clear console log now.